From 61e374f24718975e8175f048e70afaaf0c4771a9 Mon Sep 17 00:00:00 2001
From: Ismo Puustinen <ismo.puustinen@intel.com>
Date: Sat, 13 May 2023 07:07:35 +0300
Subject: [PATCH 50/79] Remove __DATE__ from version strings (#198)

Having __DATE__ is causing compilation errors when it's defined by Bazel
build tool like this: -D__DATE__=redacted
---
 frontends/tasm/tasm.c     | 1 -
 frontends/vsyasm/vsyasm.c | 1 -
 frontends/yasm/yasm.c     | 1 -
 3 files changed, 3 deletions(-)

diff --git a/frontends/tasm/tasm.c b/frontends/tasm/tasm.c
index 93c11c98..353f8c58 100644
--- a/frontends/tasm/tasm.c
+++ b/frontends/tasm/tasm.c
@@ -228,7 +228,6 @@ static opt_option options[] =
 /* version message */
 /*@observer@*/ static const char *version_msg[] = {
     PACKAGE_STRING,
-    "Compiled on " __DATE__ ".",
     "Copyright (c) 2001-2010 Peter Johnson and other Yasm developers.",
     "Run yasm --license for licensing overview and summary."
 };
diff --git a/frontends/vsyasm/vsyasm.c b/frontends/vsyasm/vsyasm.c
index 892b9b33..b3a5a592 100644
--- a/frontends/vsyasm/vsyasm.c
+++ b/frontends/vsyasm/vsyasm.c
@@ -220,7 +220,6 @@ static opt_option options[] =
 /* version message */
 /*@observer@*/ static const char *version_msg[] = {
     PACKAGE_STRING,
-    "Compiled on " __DATE__ ".",
     "Copyright (c) 2001-2010 Peter Johnson and other Yasm developers.",
     "Run yasm --license for licensing overview and summary."
 };
diff --git a/frontends/yasm/yasm.c b/frontends/yasm/yasm.c
index bc6f379a..7ac141e6 100644
--- a/frontends/yasm/yasm.c
+++ b/frontends/yasm/yasm.c
@@ -219,7 +219,6 @@ static opt_option options[] =
 /* version message */
 /*@observer@*/ static const char *version_msg[] = {
     PACKAGE_STRING,
-    "Compiled on " __DATE__ ".",
     "Copyright (c) 2001-2014 Peter Johnson and other Yasm developers.",
     "Run yasm --license for licensing overview and summary."
 };
-- 
2.49.0

